src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/dto/TestResultsDTO.java

```diff
@@ -37,7 +37,7 @@ public static TestResultsDTO convert(Object someResult) {
         return new ObjectMapper().registerModule(new JavaTimeModule()).convertValue(someResult, TestResultsDTO.class);
     }
 
-    public int getSuccessful() {
+    public Integer getSuccessful() {
```

Comment:
was this intentionally changed to `Integer`?

Reply:
I guess it can stay as int

Reply:
We needed this so we can use `.doubleValue`. I changed that place to a `(double)` cast so we can use `int` here.